constraint/pkg/apis/templates/yaml_constant.go

```diff
@@ -16,7 +17,6 @@ spec:
     listKind: ConstraintTemplateList
     plural: constrainttemplates
     singular: constrainttemplate
-  preserveUnknownFields: false
```

Comment:
we probably want to keep the setting of `preserveUnknownFields` to `false`

Reply:
yea this is what i mentioned https://github.com/open-policy-agent/frameworks/pull/134#discussion_r694414463 here, checking why this is showing as diff

Reply:
@julianKatz This is where we're losing `preserveUnknownFields`

Reply:
do we still need this? looks like it might be deprecated in v1 CRD https://github.com/kubernetes/apiextensions-apiserver/blob/master/pkg/apis/apiextensions/v1/types.go#L68

Reply:
We do still need it, otherwise existing users may still have `PreserveUnknownFields` set to `true`. It was actually added as part of the v1 CRD migration.

Reply:
isn't it noop if it's set to true in v1? https://github.com/kubernetes/apiextensions-apiserver/blob/release-1.21/pkg/apis/apiextensions/validation/validation.go#L1442-L1445

Reply:
added steps to generate `spec.preserveUnknownFields`